browser/widevine/BUILD.gn

```diff
@@ -50,43 +50,75 @@ source_set("constants") {
 source_set("unittest") {
   testonly = true
 
-  sources = [ "widevine_cdm_component_installer_unittest.cc" ]
+  sources = []
   deps = [
+    "//base",
```

Comment:
There shouldn't be any deps at all if there are no sources. I think it's best to split these tests into two separate source_sets and them as deps `:component_installer_unittest` and `:widevine_permission_android_unittest`

Reply:
It is splitted into two parts now.